java/source/org/libreoffice/ide/eclipse/java/export/AntScriptExportWizardPage.java

```diff
@@ -0,0 +1,266 @@
+/*************************************************************************
+ *
+ * The Contents of this file are made available subject to the terms of
+ * the GNU Lesser General Public License Version 2.1
+ *
+ * GNU Lesser General Public License Version 2.1
+ * =============================================
+ * Copyright 2009 by Cédric Bosdonnat
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License version 2.1, as published by the Free Software Foundation.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+ * MA 02111-1307 USA
+ *
+ * The Initial Developer of the Original Code is: Cédric Bosdonnat.
+ *
+ * Copyright: 2009 by Cédric Bosdonnat
+ *
+ * All Rights Reserved.
+ *
+ ************************************************************************/
+package org.libreoffice.ide.eclipse.java.export;
+
+import java.io.File;
+
+import org.eclipse.core.resources.IProject;
+import org.eclipse.core.resources.IResource;
+import org.eclipse.core.resources.ResourcesPlugin;
+import org.eclipse.core.runtime.CoreException;
+import org.eclipse.jface.wizard.WizardPage;
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.events.ModifyEvent;
+import org.eclipse.swt.events.ModifyListener;
+import org.eclipse.swt.layout.GridData;
+import org.eclipse.swt.layout.GridLayout;
+import org.eclipse.swt.widgets.Combo;
+import org.eclipse.swt.widgets.Composite;
+import org.eclipse.swt.widgets.Label;
+import org.libreoffice.ide.eclipse.core.PluginLogger;
+import org.libreoffice.ide.eclipse.core.gui.PackageContentSelector;
+import org.libreoffice.ide.eclipse.core.model.IUnoidlProject;
+import org.libreoffice.ide.eclipse.core.model.ProjectsManager;
+import org.libreoffice.ide.eclipse.core.model.language.LanguageExportPart;
+import org.libreoffice.ide.eclipse.java.Messages;
+import org.libreoffice.plugin.core.model.UnoPackage;
+
+public class AntScriptExportWizardPage extends WizardPage {
+
+    private IUnoidlProject sSelectedProject;
+    private LanguageExportPart sLangPart;
```

Comment:
Hm this LanguageExportPart was designed to add some language specific options to the package export wizard.
It seems a bit weird that we have a LangaugeExportPart on a java specific AntScriptWizardPage.
Maybe you can rename it so it matches better what it's actually used for now? Also I don't know if we still need to derive from the LanguageExportPart, or we just make it an own class?
Maybe also just include the relevant stuff directly here without this extra class?

Not sure what is best, but this looks cumbersome at least.